Generation/doc/AddingPileupToGeneration.md

```diff
@@ -0,0 +1,33 @@
+
+
+# Adding Pileup in FCCSW in Generation
```

Comment:
I'd rather create one file which lists all options for Pile up handling and where things can be mixed. And then refer to this file.